src/main.ts

```diff
@@ -22,19 +23,18 @@ async function run(): Promise<void> {
 
     const issueKey = githubConnector.getIssueKeyFromTitle();
 
-    if (!issueKey) {
-      console.log(`JIRA key was not found`);
-      process.exit(0);
-    }
+    if (!issueKey) throw core.error;
 
     console.log(`JIRA key -> ${issueKey}`);
 
-    const details = await jiraConnector.getTicketDetails(issueKey);
-    await githubConnector.updatePrDetails(details);
+    await jiraConnector.getTicketDetails(issueKey);
   } catch (error) {
-    console.log({ error });
-    core.setFailed(error.message);
-    process.exit(1);
+    console.log('JIRA key was not found');
+    if (FAIL_WHEN_JIRA_ISSUE_NOT_FOUND) {
+      console.log({ error });
+      core.setFailed(error.message);
+      process.exit(1);
+    } else process.exit(0);
```

Comment:
please don't use one-liners in conditions(I should add linter autofix here)
why?
it's processed ok by js, but
* it's harder to read cause it breaks visual rhyme
* it would produce bigger diff if second line would be added 
* it's easier to make mistake here in the future

Reply:
Updated and removed one liner